src/java.base/share/classes/jdk/internal/access/foreign/MappedMemoryUtilsProxy.java

```diff
@@ -0,0 +1,14 @@
+package jdk.internal.access.foreign;
+
+import java.io.FileDescriptor;
+
+/**
+ * This proxy interface is required to allow access to @{code MappedMemoryUtils} methods from {@code ScopedMemoryAccess}.
+ * This allows to avoid pesky initialization issues in the middle of memory mapped scoped methods.
+ */
+public interface MappedMemoryUtilsProxy {
```

Comment:
We should probably try to consolidate this and `UmapperProxy` - I think it can be done, but it requires deeper surgery than I was willing to do in this PR.

Reply:
Although this proxy class may be temporarily, I'll think you'll need to put a copyright header on it.